lib/main.coffee

```diff
@@ -14,14 +33,36 @@ module.exports =
       (executablePath) =>
         @executablePath = executablePath
 
+    # Add a listener to reload this package if the custom grammar scopes setting
+    # changes. Grammar scopes can only be set at load time.
+    @subscriptions.add atom.config.observe 'linter-tidy.customGrammarScopes',
+      () ->
```

Comment:
I'm not sure what you are doing with the reloading of the package here... there is no need for that.

This block should just be a copy of [this](https://github.com/AtomLinter/linter-htmlhint/blob/00417d8335267391920afe16b51e28431a410034/lib/index.js#L11-L16).